hack/lib/build/version.sh

```diff
@@ -72,6 +73,14 @@ function os::build::version::git_vars() {
 }
 readonly -f os::build::version::git_vars
 
+function os::build::version::buildah_vars() {
+	if [[ -n "${OS_BUILDAH_VERSION-}" ]]; then
+		return 0
+	fi
+	OS_BUILDAH_VERSION=$(go list -mod=mod -m -f '{{.Version}}' github.com/containers/buildah)
```

Comment:
We need to pass `-mod=mod` because we have a vendor directory. The format is a golang format driven by a struct.

`$ go help list` provides the specification.